mewarpx/mewarpx/mwxrun.py

```diff
@@ -28,7 +28,8 @@
 import os.path
 
 from pywarpx import _libwarpx, picmi, fields
-from mewarpx.utils_store import mwxconstants as constants, parallel_util, profileparser, init_restart_util
+from mewarpx.utils_store import mwxconstants as constants, parallel_util
```

Comment:
```suggestion
from mewarpx.utils_store import mwxconstants as constants
from mewarpx.utils_store import parallel_util
```
If we're changing it anyway - I don't really like `import ... as` in lists that include other things, because it's not all that human-readable.